src/components/Modal/DownloadModal.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useCallback } from 'react'
+
+import { Modal, ModalProps } from 'decentraland-ui/dist/components/Modal/Modal'
+import { DownloadButton } from 'decentraland-ui2/dist/components/DownloadButton/DownloadButton'
```

Comment:
We should export this as global. Nothing to do here, but to take not
```suggestion
import { DownloadButton } from 'decentraland-ui2'
```

Reply:
https://github.com/decentraland/ui2/pull/241